frontend/lib/src/components/shared/InputInstructions/InputInstructions.test.tsx

```diff
@@ -154,5 +154,18 @@ describe("InputInstructions", () => {
 
       expect(screen.getByTestId("InputInstructions")).toHaveTextContent("")
     })
+
+    it("should show enter instructions in form even when input is not dirty", () => {
+      const props = getProps({
+        dirty: false,
+        inForm: true,
+        allowEnterToSubmit: true,
+      })
+      render(<InputInstructions {...props} />)
+
+      expect(screen.getByTestId("InputInstructions").textContent).toBe(
+        "Press Enter to submit form"
+      )
```

Comment:
**suggestion:** To adhere to [RTL best practices](https://testing-library.com/docs/queries/about/#priority), utilize a `getByText` call instead. Perhaps:

```suggestion
      expect(screen.getByText('Press Enter to submit form')).toBeVisible()
```

Reply:
Makes sense. Btw that's an awesome resource!